lib/linguist/languages.yml

```diff
@@ -348,6 +348,7 @@ Apex:
   type: programming
   color: "#1797c0"
   extensions:
+  - ".apex"
```

Comment:
Is this now the language's primary extension? If not, it needs to come after `.cls` which is currently considered the primary language.

From the to of the file:

https://github.com/github-linguist/linguist/blob/39fd5e93de98de3434cb3e857c5b1972c418f8c5/lib/linguist/languages.yml#L17-L19

Reply:
It is not the primary, I will move it after